src/site/stages/build/process-cms-exports/transformers/paragraph-wysiwyg.js

```diff
@@ -0,0 +1,20 @@
+const { getDrupalValue } = require('./helpers');
+
+function wysiwygTransform(entity) {
+  const { fieldWysiwyg } = entity;
+
+  const transformed = {
+    entity: {
+      entityType: 'paragraph',
+      entityBundle: 'wysiwyg',
```

Comment:
It might be a good idea to move this into `transformEntity` and have it set the values based on parsing `entity.contentModelType` or something like that